hugegraph-server/hugegraph-dist/release-docs/licenses/LICENSE-java-cup-runtime.txt

```diff
@@ -0,0 +1,10 @@
+
+CUP Parser Generator Copyright Notice, License, and Disclaimer
+Copyright 1996-2015 by Scott Hudson, Frank Flannery, C. Scott Ananian, Michael Petter
+
+Permission to use, copy, modify, and distribute this software and its documentation for any purpose and without fee is hereby granted, provided that the above copyright notice appear in all copies and that both the copyright notice and this permission notice and warranty disclaimer appear in supporting documentation, and that the names of the authors or their employers not be used in advertising or publicity pertaining to distribution of the software without specific, written prior permission.
+
+The authors and their employers disclaim all warranties with regard to this software, including all implied warranties of merchantability and fitness. In no event shall the authors or their employers be liable for any special, indirect or consequential damages or any damages whatsoever resulting from loss of use, data or profits, whether in an action of contract, negligence or other tortious action, arising out of or in connection with the use or performance of this software.
+This is an open source license. It is also GPL-Compatible (see entry for "Standard ML of New Jersey"). The portions of CUP output which are hard-coded into the CUP source code are (naturally) covered by this same license, as is the CUP runtime code linked with the generated parser.
```

Comment:
It seems that it is not clear from the file what license this dependency is based on. And this license contains `GPL` keywords, which should be noted and removed.

Reply:
> It seems that it is not clear from the file what license this dependency is based on. And this license contains `GPL` keywords, which should be noted and removed.

consider remove the unnecessary NOTICE part from upstream